tests/integration/test_notebooks.py

```diff
@@ -224,6 +232,21 @@ def _run_notebook(
         if not (line.startswith("%") or line.startswith("!"))
     ]
 
+    # Replace config parms
+    if params:
+        transform = lambda line: line.replace(
+            "device_limit_frac = 0.7", "device_limit_frac = " + str(params[0])
+        )
+        lines = [transform(line) for line in lines]
+        transform = lambda line: line.replace(
+            "device_pool_frac = 0.8", "device_pool_frac = " + str(params[1])
+        )
+        lines = [transform(line) for line in lines]
+        transform = lambda line: line.replace(
+            "part_mem_frac = 0.15", "part_mem_frac = " + str(params[2])
+        )
+        lines = [transform(line) for line in lines]
```

Comment:
I think we don't allow assigning lambdas (flake8). Maybe something like this instead?

```suggestion
        def transform(line):
            line = line.replace("device_limit_frac = 0.7", "device_limit_frac = " + str(params[0]))
            line = line.replace("device_pool_frac = 0.8", "device_pool_frac = " + str(params[1]))
            return line.replace("part_mem_frac = 0.15", "part_mem_frac = " + str(params[2]))

        lines = [transform(line) for line in lines]
```

Reply:
Got it! Thanks Ben!